lib/aws/lex/conversation/type/session_attributes.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Aws
+  module Lex
+    class Conversation
+      module Type
+        class SessionAttributes < Hash
+          include Base
+
+          optional :checkpoints
+
+          def checkpoints
+            @checkpoints ||= begin # rubocop:disable Style/RedundantBegin
```

Comment:
I'm seeing a rubocop warning here:

```
lib/aws/lex/conversation/type/session_attributes.rb:13:36: W: [Correctable] Lint/RedundantCopDisableDirective: Unnecessary disabling of Style/RedundantBegin.
```

Reply:
interesting! let me double-check that.

Reply:
Just a note that I may have a different version of rubocop installed locally - I'm not sure if we're locking it down.

Reply:
CI is passing, so that should be the final say really.